templates/docs/api.html

```diff
@@ -4,6 +4,15 @@
 {% block page_title %}| API {% endblock %}
 
 {% block content %}
+  <section id="search-docs" class="p-strip--image is-shallow" style="background-image: url('https://assets.ubuntu.com/v1/e54487e2-maas-docs-suru.png')">
```

Comment:
These inline styles could be added to css ideally.

Reply:
I think I'll just leave them as inline for now if it's all the same to you.

We can have a discussion about inline styles in Dev catch-up if you like.